apis/zones/zones_addrecordset.go

```diff
@@ -19,3 +20,16 @@ func (c *client) AddRecordSetToZone(ctx context.Context, serverID string, zoneID
 
 	return c.httpClient.Patch(ctx, path, nil, pdnshttp.WithJSONRequestBody(&patch))
 }
+
+func (c *client) AddRecordSetsToZone(ctx context.Context, serverID string, zoneID string, sets []ResourceRecordSet) error {
+	path := fmt.Sprintf("/servers/%s/zones/%s", url.PathEscape(serverID), url.PathEscape(zoneID))
+
+	for idx := range sets {
+		sets[idx].ChangeType = ChangeTypeReplace
+	}
+	patch := Zone{
+		ResourceRecordSets: sets,
+	}
+
+	return c.httpClient.Patch(ctx, path, nil, pdnshttp.WithJSONRequestBody(&patch))
```

Comment:
To prevent the duplicated code (now, the implementation of `AddRecordSetToZone` and `AddRecordSetsToZone` is largely identical), maybe we could refactor `AddRecordSetToZone` to invoke `AddRecordsSetToZone` under the hood? 🤔 

Thinking a bit further, it might also make sense to deprecate `AddRecordSetToZone`, since there's nothing you can do with it that you couldn't also do with `AddRecordSetsToZone`. WDYT?

Reply:
> maybe we could refactor `AddRecordSetToZone` to invoke `AddRecordsSetToZone` under the hood?

Entirely possible. It is a simple change to make. Would you like me to do that?

> Thinking a bit further, it might also make sense to deprecate `AddRecordSetToZone`, since there's nothing you can do with it that you couldn't also do with `AddRecordSetsToZone`. WDYT?

I think that's up to you. 🙂 It does make sense though with the above change.

Reply:
I hadn't seen a reply yet so I just made the change for review. We can remove it if it's not wanted.

Reply:
Oh, apologies. 🙏  Thanks for the update. 😃